srcpkgs/saw/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'saw'
+pkgname=saw
+version=0.2.2
+revision=1
+archs="i686 x86_64"
```

Comment:
Is there a reason this doesn't work on ARM architectures? It appears to compile fine.

Reply:
I didn't realize this field was optional, removed